### PR TITLE
chia: 1.3.1 -> 1.5.0

### PR DESCRIPTION
**Pull request by bbjubjub2494**
###### Description of changes

https://github.com/Chia-Network/chia-blockchain/releases/tag/1.5.0

###### Things done

- Built on platform(s)
  - [x] x86_64-linux
  - [ ] aarch64-linux
  - [ ] x86_64-darwin
  - [ ] aarch64-darwin
- [ ] For non-Linux: Is `sandbox = true` set in `nix.conf`? (See [Nix manual](https://nixos.org/manual/nix/stable/command-ref/conf-file.html))
- [x] Tested, as applicable:
  - [NixOS test(s)](https://nixos.org/manual/nixos/unstable/index.html#sec-nixos-tests) (look inside [nixos/tests](https://github.com/NixOS/nixpkgs/blob/master/nixos/tests))
  - and/or [package tests](https://nixos.org/manual/nixpkgs/unstable/#sec-package-tests)
  - or, for functions and "core" functionality, tests in [lib/tests](https://github.com/NixOS/nixpkgs/blob/master/lib/tests) or [pkgs/test](https://github.com/NixOS/nixpkgs/blob/master/pkgs/test)
  - made sure NixOS tests are [linked](https://nixos.org/manual/nixpkgs/unstable/#ssec-nixos-tests-linking) to the relevant packages
- [x] Tested compilation of all packages that depend on this change using `nix-shell -p nixpkgs-review --run "nixpkgs-review rev HEAD"`. Note: all changes have to be committed, also see [nixpkgs-review usage](https://github.com/Mic92/nixpkgs-review#usage)
- [ ] Tested basic functionality of all binary files (usually in `./result/bin/`)
- [22.11 Release Notes (or backporting 22.05 Release notes)](https://github.com/NixOS/nixpkgs/blob/master/CONTRIBUTING.md#generating-2211-release-notes)
  - [ ] (Package updates) Added a release notes entry if the change is major or breaking
  - [ ] (Module updates) Added a release notes entry if the change is significant
  - [ ] (Module addition) Added a release notes entry if adding a new NixOS module
  - [ ] (Release notes changes) Ran `nixos/doc/manual/md-to-db.sh` to update generated release notes
- [x] Fits [CONTRIBUTING.md](https://github.com/NixOS/nixpkgs/blob/master/CONTRIBUTING.md).

### CODE REVIEW

**Comment by lourkeur** [comment]
partial redundancy with 3ef7621167df207d13fbe288b78cb9f8573bc465

File: pkgs/development/python-modules/clvm-tools-rs/default.nix [hunk 1]
```diff
@@ -0,0 +1,41 @@
...
+  };
+
+  cargoDeps = rustPlatform.fetchCargoTarball {
+    inherit src;
+    name = "${pname}-${version}";
+    sha256 = "sha256-vSQEkhqt3ts0+6INB+JNRDi+nZHiHW3r1vSb2GYNMiM=";
+  };
+
+  format = "pyproject";
+
```
Comment (SuperSandro2000) [inline pkgs/development/python-modules/clvm-tools-rs/default.nix:27]:
```suggestion
```

File: pkgs/development/python-modules/clvm-tools-rs/default.nix [hunk 2]
```diff
@@ -0,0 +1,41 @@
...
+, fetchFromGitHub
+, buildPythonPackage
+, rustPlatform
+, pythonOlder
+}:
+
+buildPythonPackage rec {
+  pname = "clvm_tools_rs";
+  version = "0.1.19";
+  disabled = pythonOlder "3.7";
```
Comment (SuperSandro2000) [inline pkgs/development/python-modules/clvm-tools-rs/default.nix:11]:
```suggestion
  disabled = pythonOlder "3.7";
  format = "pyproject";
```

File: pkgs/development/python-modules/clvm-tools-rs/default.nix [hunk 3]
```diff
@@ -0,0 +1,41 @@
+{ lib
+, fetchFromGitHub
+, buildPythonPackage
+, rustPlatform
+, pythonOlder
+}:
+
+buildPythonPackage rec {
+  pname = "clvm_tools_rs";
```
Comment (SuperSandro2000) [inline pkgs/development/python-modules/clvm-tools-rs/default.nix:9]:
```suggestion
  pname = "clvm-tools-rs";
```

File: pkgs/development/python-modules/clvm-tools-rs/default.nix [hunk 4]
```diff
@@ -0,0 +1,41 @@
...
+  version = "0.1.19";
+  disabled = pythonOlder "3.7";
+
+  src = fetchFromGitHub {
+    owner = "Chia-Network";
+    repo = pname;
```
Comment (SuperSandro2000) [inline pkgs/development/python-modules/clvm-tools-rs/default.nix:15]:
```suggestion
    repo = "clvm_tools_rs";;
```

File: pkgs/development/python-modules/chia-rs/default.nix [hunk 5]
```diff
@@ -0,0 +1,49 @@
+{ buildPythonPackage
+, lib
+, fetchFromGitHub
+, pytestCheckHook
+, rustPlatform
+}:
+
+buildPythonPackage rec {
+  pname = "chia_rs";
```
Comment (SuperSandro2000) [inline pkgs/development/python-modules/chia-rs/default.nix:9]:
```suggestion
  pname = "chia-rs";
```

File: pkgs/development/python-modules/chia-rs/default.nix [hunk 6]
```diff
@@ -0,0 +1,49 @@
...
+  version = "0.1.5";
+
+  src = fetchFromGitHub {
+    owner = "chia-network";
+    repo = pname;
```
Comment (SuperSandro2000) [inline pkgs/development/python-modules/chia-rs/default.nix:14]:
```suggestion
    repo = "chia_rs";;
```

File: pkgs/development/python-modules/chia-rs/default.nix [hunk 7]
```diff
@@ -0,0 +1,49 @@
...
+  ];
+
+  buildAndTestSubdir = "wheel";
+
+  preBuild = ''
+    # avoid ENOENT in maturinBuildHook
+    touch wheel/Cargo.lock
+  '';
+
+  meta = with lib; {
```
Comment (SuperSandro2000) [inline pkgs/development/python-modules/chia-rs/default.nix:43]:
```suggestion
  meta = with lib; {
```

File: pkgs/development/python-modules/chia-rs/default.nix [hunk 8]
```diff
@@ -0,0 +1,49 @@
...
+
+  postPatch = ''
+    cp ${./Cargo.lock} Cargo.lock
+  '';
+
+  nativeBuildInputs = with rustPlatform; [
+    cargoSetupHook
+    maturinBuildHook
+  ];
+
```
Comment (SuperSandro2000) [inline pkgs/development/python-modules/chia-rs/default.nix:31]:
```suggestion

  preBuild = ''
    # avoid ENOENT in maturinBuildHook
    touch wheel/Cargo.lock
  '';

```

File: pkgs/development/python-modules/chia-rs/default.nix [hunk 9]
```diff
@@ -0,0 +1,49 @@
...
+  buildAndTestSubdir = "wheel";
+
+  preBuild = ''
+    # avoid ENOENT in maturinBuildHook
+    touch wheel/Cargo.lock
+  '';
+
+  meta = with lib; {
+    description = "Rust crate & wheel with consensus code";
+    homepage = "https://www.chia.net/";
```
Comment (SuperSandro2000) [inline pkgs/development/python-modules/chia-rs/default.nix:45]: Could we link to something specific to this package instead of the super generic homepage?

Comment (lourkeur) [reply pkgs/development/python-modules/chia-rs/default.nix:45]: Not really. It's first-party and very generic. There's the [GH page](https://github.com/Chia-Network/chia_rs/), but all it says is:

> This cargo workspace contains code useful for working with the Chia network.
>
> It contains a rust crate chia (here), a python wheel api chia_rs (in wheel), and web assembly (in wasm).

Comment (SuperSandro2000) [reply pkgs/development/python-modules/chia-rs/default.nix:45]: That would be better especially from a reviewer perspective.

Comment (lourkeur) [reply pkgs/development/python-modules/chia-rs/default.nix]: Fixed

**Pull request merged**